src/web/crate_details.rs

```diff
@@ -531,4 +546,64 @@ mod tests {
             Ok(())
         })
     }
+
+    #[test]
+    fn test_latest_version_ignores_yanked() {
+        crate::test::wrapper(|env| {
+            let db = env.db();
+
+            db.fake_release().name("foo").version("0.0.1").create()?;
+            db.fake_release()
+                .name("foo")
+                .version("0.0.3")
+                .cratesio_data_yanked(true)
```

Comment:
I think this should be named `yanked()` instead to better fit the other functions, are you interested in doing that here? If not I can make a follow-up PR.

Reply:
I was _so close_ to making that change, then thought I'd avoid extraneous changes 😄